src/components/reusable/stepper/stepperItem.scss

```diff
@@ -0,0 +1,292 @@
+    flex: 1 1 auto;
+    height: 1px;
+    overflow: hidden;
+    right: calc(-50% + 20px);
+    left: calc(50% + 20px);
+    position: absolute;
+    background-color: var(--kd-color-background-inverse-hover, #ddf0f3);
+    border-radius: 5px;
+    &-large {
+      top: 11px;
+    }
+    &-small {
+      top: 8px;
+    }
+    &-first {
+      left: 30px;
+    }
+    &-secondlast {
+      right: calc(-100% + 30px);
+    }
+  }
+}
+
+.progressbar {
+  top: 0;
+  left: 0;
+  width: 0;
+  bottom: 0;
+  position: absolute;
+  transition: transform 0.2s linear;
+  transform-origin: left;
+  background-color: var(--kd-color-border-accent-spruce-light, #3fadbd);
+  &-completed {
+    background-color: var(--kd-color-spruce-50);
+  }
+}
+
+.step {
+  &-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 4px;
+    &-first {
+      align-items: flex-start;
+    }
+    &-last {
+      align-items: flex-end;
+    }
+  }
+
+  &-text {
+    margin-top: 8px;
+    color: var(--kd-color-text-primary);
+    font-family: Roboto;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    text-transform: uppercase;
+    &-disabled {
+      cursor: not-allowed;
+      color: var(--kd-color-darkstone-40);
+      pointer-events: none;
+    }
+    &-error {
+      color: var(--kd-color-error);
+    }
+  }
+
+  &-title-wrapper {
+    display: flex;
+    gap: 4px;
+    align-items: center;
+  }
+
+  &-title {
+    margin-top: 4px;
+  }
+
+  &-title-text {
+    color: var(--kd-color-text-secondary);
+    font-family: Roboto;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px;
+    letter-spacing: 0.16px;
+  }
+}
+
+/** vertical stepper styles */
+
+.vertical {
+  &-wrapper {
+    display: inline-flex;
+    width: 100%;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  &-step {
+    min-width: 174px;
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    position: relative;
+  }
+
+  &-stepper-container {
+    padding-bottom: 20px;
+    position: relative;
+    min-height: 50px;
+    display: flex;
+    &-large {
+      padding-left: 32px;
+    }
+    &-small {
+      padding-left: 24px;
+    }
+  }
+
+  &-stepper-line {
+    position: absolute;
+    border-left-style: solid;
+    border-left-width: 1px;
+    bottom: 0;
+    border-color: var(--kd-color-background-inverse-hover, #ddf0f3);
+    border-radius: 5px;
+    &-large {
+      top: 27px;
+      left: 11.5px;
+    }
+    &-small {
+      left: 8px;
+      top: 19px;
+    }
+  }
+
+  &-progress-line {
+    top: 0;
+    height: 0;
+    left: auto;
+    right: 0;
+    bottom: 0;
+    position: absolute;
+    border-left-style: solid;
+    border-left-width: 1px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    border-color: var(--kd-color-border-accent-spruce-light, #3fadbd);
+    &-completed {
+      border-color: var(--kd-color-spruce-50);
```

Comment:
Can't use palette color tokens directly, must be a named token.

Reply:
Done